src/lib.rs

```diff
@@ -452,6 +597,12 @@ impl<D, T> Message<D, T> {
             .duration_since(UNIX_EPOCH)
             .expect("Time went backwards");
 
+        let topic = match (hedwig.message_router)(&data_type, &data_schema_version.0) {
```

Comment:
do you need the () around `hedwig.message_router`

Reply:
yeah that was odd.. 
```
help: to call the function stored in `message_router`, surround the field access with parentheses
```